build/stage-release-googleplay.yml

```diff
@@ -4,11 +4,11 @@ parameters:
 
 jobs:
 - deployment:  GooglePlay_Android
-  pool: $(windowsPoolName)
 
-  environment: ${{ parameters.deploymentEnvironment }}
+  pool:
+    vmImage: $(windowsHostedAgentImage)
```

Comment:
isn't google play for android ? Why does it have windows in its name

Reply:
Using a Windows agent to push it to the store. It's the machine's OS.